src/Program.cs

```diff
@@ -33,11 +33,11 @@ private static async Task<int> Main(string[] args)
                 {
                     Argument = new Argument<string>(() => null)
                 },
-                new Option(new[] { "--include", "--files" }, Resources.A_comma_separated_list_of_relative_file_or_folder_paths_to_include_in_formatting_All_files_are_formatted_if_empty)
+                new Option(new[] { "--include", "--files" }, Resources.A_list_of_relative_file_or_folder_paths_to_include_in_formatting_All_files_are_formatted_if_empty)
                 {
                     Argument = new Argument<string>(() => null)
```

Comment:
@JoeRobich, this was changed to `string[]` and also the `--exclude` case below.

Reply:
That is what I get for trying to fix merge conflicts using the GH editor. sorry

Reply:
No problem, thanks for the merge. I think API should error out in this case. Opened https://github.com/dotnet/command-line-api/issues/787 for discussion.